src/Helper/MyParcelCollection.php

```diff
@@ -300,6 +305,18 @@ public function createConcepts(): self
         return $this;
     }
 
+    /**
+     * Shipment v2 endpoint active from x number of orders
+     *
+     * @param int $numberOfOrders
+     *
+     * @return bool
+     */
+    public function useShipmentV2(int $numberOfOrders): bool
```

Comment:
Het is voor andere programmeurs niet duidelijk wat je bedoeld moet v2. Het is eigenlijk ook niet goed dat we dit V2 noemen. Je kan dit beter preparePdf noemen (of iets anders waardoor andere weten wat je bedoeld). Ook in andere plekken in de sdk aanpassen :), Ik zie ook dat je die const setup label noemt, dat is ook een mooie naam.

Reply:
Done